db/db_iter.cc

```diff
@@ -681,6 +681,7 @@ bool DBIter::ReverseToForward() {
     last_key.SetInternalKey(ParsedInternalKey(
         saved_key_.GetUserKey(), kMaxSequenceNumber, kValueTypeForSeek));
     iter_.Seek(last_key.GetInternalKey());
+    RecordTick(statistics_, NUMBER_OF_RESEEKS_IN_ITERATION);
```

Comment:
Should we update `ReverseToBackward` as well?

Reply:
We can. I didn't not include `ReverseToBackward` because it involves Merge, and I intentionally wants to keep the scope of a single PR as small as possible. Maybe I can open a new PR for that later.

Reply:
Let me update it and see if it can remain small.

Reply:
@ltamasi , fortunately the new update does not break any existing test. Therefore, I added a few lines to test it.